source/extensions/stat_sinks/common/statsd/statsd.h

```diff
@@ -14,6 +14,7 @@ namespace StatSinks {
 namespace Common {
 namespace Statsd {
 
+static const std::string DEFAULT_PREFIX = "envoy";
```

Comment:
This should use CONSTRUCT_ON_FIRST_USE so should be a function

Reply:
Replaced with `static const std::string& getDefaultPrefix()`